examples/multi-account-multi-region/scanning_account.tf

```diff
@@ -31,7 +31,7 @@ module "lacework_aws_agentless_scanning_global" {
     // This account must also have the snapshot_role installed.
     management_account = "0001234567890"
   }
-
```

Comment:
Would recommend adding an example with the account mapping

Reply:
Added a separate example calling out differences b/t regular multi-account deployment & lw org deployment